pages/VerifiableCredentials/Overview.md

```diff
@@ -0,0 +1,75 @@
+# Verifiable Credentials Specification
+__Version pre-1.3.0__
+
+[Attribute Sets](../DSNP/AttributeSets.md) and [Attestation](../ActivityContent/Associated/Attachments.md#attestation) attachments shared via DSNP reference data documents containing Verifiable Credentials and related objects.
+For DSNP purposes, certain restrictions and extensions are applied to the base World Wide Web Consortium (W3C) specification documents noted below.
+When there are DSNP extensions, they are guaranteed to use non-colliding terms.
+
+It is anticipated that these W3C specifications will evolve, and applications interacting with Verifiable Credentials, Schemas, and DIDs via DSNP are encouraged to be as flexible as possible in their syntax support as new versions and supporting software libraries become available.
+Therefore, compatibility with the following versions and specifications should be construed as the minimal level of support required to ensure interoperability between DSNP applications.
+
+## Compatibility
+
+Current usage with DSNP relies on the following specifications:
+
+| Specification | Version/Status | Relevant JSON-LD `@context` Values |
+| --- | --- | --- | 
+ | Verifiable Credentials Data Model | [1.1 (W3C Recommendation)](https://www.w3.org/TR/vc-data-model-1.1)<br>[2.0 (W3C Candidate Recommendation Draft 23 August 2024)](https://www.w3.org/TR/2024/CRD-vc-data-model-2.0-20240823/) | `https://www.w3.org/2018/credentials/v1`<br>`https://www.w3.org/ns/credentials/v2` |
+ | Verifiable Credential Data Integrity | [1.0 (W3C Candidate Recommendation Draft 3 August 2024)](https://www.w3.org/TR/2024/CRD-vc-data-integrity-20240803/) | `https://w3id.org/security/data-integrity/v2` |
+ | Verifiable Credentials JSON Schema | [(W3C Candidate Recommendation Draft  18 December 2023)](https://www.w3.org/TR/2023/CRD-vc-json-schema-20231218/) | |
+ | Decentralized Identifiers (DIDs) | [1.0 (W3C Recommendation 19 July 2022)](https://www.w3.org/TR/2022/REC-did-core-20220719/) | `https://www.w3.org/ns/did/v1` |
+
+### Cryptography
+
+The Data Integrity specification provides a generic format for expressing cryptographic proofs, where the detailed representation of each data item is defined in individual cryptosuites.
+DSNP compliant applications MUST support the following cryptosuites, which correspond to the allowed algorithms for `assertionMethod` [Public Keys](../DSNP/Types/PublicKey.md):
+
+| Specification | Version/Status | Multikey codec |
+| --- | --- | --- |
+| Data Integrity EdDSA Cryptosuites | [1.0 (W3C Candidate Recommendation Draft 16 August 2024)](https://www.w3.org/TR/2024/CRD-vc-di-eddsa-20240816/) | `ed25519-pub` |
```

Comment:
When I follow this link I get the following message on the site:
This version is outdated!
For the latest version, please look at [https://www.w3.org/TR/vc-di-eddsa/.](https://www.w3.org/TR/vc-di-eddsa/)

Reply:
Updated